src/front/ticketer/lib/pages/common/password_change_dialog.dart

```diff
@@ -0,0 +1,185 @@
+import 'package:flutter/material.dart';
+import 'package:ticketer/auth/auth.dart';
+
+Future<void> changePasswordDialog(BuildContext context) async {
+  await showDialog(
+    context: context,
+    builder: (context) {
+      return AlertDialog(
+        title: const Text("Change password"),
+        content: PasswordEntry(),
+        actions: [
+          ElevatedButton(
+            onPressed: () => {
+              Navigator.pop(context),
+            },
+            child: const Text('OK'),
```

Comment:
suggestion: maybe change label to "Cancel" and change colour to redAccent?

Reply:
<img width="383" alt="image" src="https://github.com/pw-software-engineering/2023-konfetti/assets/73365815/3920bf60-38fe-44bc-a0e4-2a7437280008">
Added